rs/tests/testing_verification/ii_delegation_test.rs

```diff
@@ -46,8 +47,10 @@ pub fn config(env: TestEnv) {
 pub fn test(env: TestEnv) {
     let log = env.logger();
     let non_nns_node = env.get_first_healthy_system_but_not_nns_node_snapshot();
-    let ii_canister_id =
-        non_nns_node.create_and_install_canister_with_arg(INTERNET_IDENTITY_WASM, None);
+    let ii_canister_id = non_nns_node.create_and_install_canister_with_arg(
+        &env::var("II_WASM_PATH").expect("II_WASM_PATH not set"),
```

Comment:
Is `II_WASM_PATH` already set somewhere else?

Reply:
@cgundy what do you mean?

Reply:
Oh I see. Yes, it's set in a couple places but wasn't used here:
* https://github.com/dfinity/ic/blob/62da1512abc7c55e6aa32f9c13df2c8f2feded74/rs/tests/crypto/BUILD.bazel#L34
* https://github.com/dfinity/ic/blob/62da1512abc7c55e6aa32f9c13df2c8f2feded74/rs/tests/nns/BUILD.bazel#L67